components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -779,6 +781,26 @@ export default class PluginManagement extends AdminSettings {
                             setByEnv={this.isSetByEnv('PluginSettings.Enable')}
                         />
 
+                        <BooleanSetting
+                            id='enableHealthCheck'
+                            disabled={!enable}
+                            label={
+                                <FormattedMessage
+                                    id='admin.plugins.settings.enableHealthCheck'
+                                    defaultMessage='Enable Health Check: '
+                                />
+                            }
+                            helpText={
+                                <FormattedMessage
+                                    id='admin.plugins.settings.enableHealthCheckDesc'
+                                    defaultMessage='When true, enables health checks on all active server plugins. If a plugin has crashed or is deemed unhealthy, the server will restart the plugin. If a plugin has crashed three times in one hour, the server will deactivate the plugin.'
```

Comment:
```suggestion
                                    defaultMessage='When true, enables health checks on all active server plugins. If a plugin has crashed or is deemed unhealthy, the server will restart the plugin. If a plugin has crashed three times within an hour, the server will deactivate the plugin.'
```

Reply:
I like that it's specific.

If we link to docs and we update the information in future, those running an old Mattermost version will be linked to a doc that has new info not relevant to them (and vice versa, new Mattermost versions may be linked to a doc with old info).

Reply:
@jasonblais 

I agree with the wording change. Re-reading it, an admin may think that it must be within an hour of server start or something similar, when in reality it is a sliding window as the server continues to run. Just a thought, probably best to keep as is for simplicity.

Great point about version mismatches. I agree with the decision of keeping it in-app.

Reply:
Yeah, fair point. Let's keep the text as-is 👍

Reply:
Sorry, I was a bit confusing with my explanation on that. By "as-is" I was implying that we would include your suggestion for the more formal wording, but not go further to explain the sliding window portion. This is really semantics, so I'm fine with any of the mentioned scenarios.

If I were to include the sliding window explanation, I would probably word it as "within a sliding one-hour window". I don't want to make a big deal about this, especially since you say it's non-blocking, though I'd like to get an opinion from a dev on the team on the wording.

Reply:
Ah, got it. I'll unresolve the conversation if others have thoughts.